app/views.py

```diff
@@ -98,7 +100,7 @@ class Meta:
 class ContractUpdate(UpdateView):
     template_name = "app/update_contract.html"
     model = Contract
-    fields = ["organizer_account_name", "organizer_email", "signed_date", "event_id", "user_id"]
+    fields = ["organizer_account_name", "organizer_email", "signed_date", "user_id"]
```

Comment:
A general comment. What's the code style for this project or for your team? Bc I'm seeing different usages of single comma and double comma across the whole project. One way to fix this is to automate these tasks with a tool/library. 
Black is a format tool that you can add to your project. The idea behind this is hit and run. Black will we in charge of auto-format your file/folder for you and you don't pay attention for code styling.
https://github.com/psf/black

(Also you can configure this using a `pre-commit` tool https://pre-commit.com/)

Reply:
We don't have defined code styles. Is a very good idea use this tool. Thanks.